### ISSUE TITLE
After add child it would be cool if I could write full sentences (with spaces) and org-brain would later replace by _ for me.

### ISSUE THREAD

**Issue by michaelsjackson** [Kungsgeten/org-brain#368]
This would simplify writing full sentences as new child entries for example.

Use case scenario: You are a philosopher or researcher from social sciences. You want to organize your full sentences, claims, arguments and so on using org-brain. Thus you write full sentences rather than a few words only.

**Comment by Kungsgeten** [comment]
Entries already support spaces (but not linebreaks) in their names. You should be able to do what you suggest.

**Comment by michaelsjackson** [comment]
Not really somehow. Before it was not working at all. After setting that mentioned parameter to t somehow org-brain allowing me using spaces, great, but while writing sentence it just jumps in and takes some action before me finishing my sentence. So there might be some hidden bugs.

**Comment by michaelsjackson** [comment]
I checked again, if I write my sentence with spaces org-brain is trying to match to available entries, not allowing me write a new sentence.

On the other hand using _ instead of space I can write any new sentence I want. This behaviour option I would like to have also with spaces. I hope it is clear now. Maybe there is another magical parameter which I need to set?

**Comment by michaelsjackson** [comment]
Also:  I do not want to have spaces in file names. Only I would like typing spaces, but org-brain would use while writing on hard disk _ instead of space. 
Typing: space
Writing to harddisk / saving as: _

**Comment by Kungsgeten** [comment]
My guess is that the issues of "jumping in" has to do with the completion system (ivy, helm, ido, etc) you're using, and not org-brain. To replace spaces with underscores seems quite user specific (and perhaps OS dependent). Perhaps it could be done, but I'm not sure if that kind of functionality is something that the package should handle.

**Comment by michaelsjackson** [comment]
Yes, could be. I like the one I am using, show as list view below. Not sure even what is its name. :-)

Which one are you using and where spaced typing is possible but org-brain would still save with _ instead. Or is this feature already included or not available yet? User specific yes, but I guess this can be useful for anyone.
1. I want to type full sentences, looking normal, thus I need spaces.
2. I want to save on hard disk replacing spaces with _ because I am also other tools like latex, pdf export or grep and so on, I do not want possible trouble in future because of those spaces. Thus replacing by _ is a security measure that it should work anywhere, without troubles. I am using many more tools which can have troubles with spaces in names or paths. 

Thanks for your response. Maybe some more users show interest in this feature.

**Comment by michaelsjackson** [comment]
Ah, my OS is ubuntu 18.04.
